namesys/publisher.go

```diff
@@ -356,9 +356,9 @@ func InitializeKeyspace(ctx context.Context, ds dag.DAGService, pub Publisher, p
 	return nil
 }
 
-func IpnsKeysForID(id peer.ID) (name, ipns key.Key) {
-	namekey := key.Key("/pk/" + id)
-	ipnskey := key.Key("/ipns/" + id)
+func IpnsKeysForID(id peer.ID) (name, ipns string) {
+	namekey := "/pk/" + string(id)
+	ipnskey := "/ipns/" + string(id)
```

Comment:
In:

- core/commands/dht_test.go:12
- namesys/publisher.go:68
- namesys/publisher.go:141
- namesys/republisher/repub.go:90